tools/evict-shared-cache/eviction-policy.h

```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+#include <string>
+
+struct EvictionPolicy {
```

Comment:
Need to flesh out the actual parameters here

Reply:
I think initially just having the job id is sufficient.

Reply:
What do you think these would be returning?

Reply:
I didn't have anything specific in mind. Maybe a simple success/fail is sufficient. Though maybe that isn't even all that valuable. Not sure how wake recovers from a eviction failure

Reply:
Yeah I think on eviction failure we should just not care. Unless the eviction processes closes the pipe we should assume its still responding even if it can't delete that one thing.